{"name": "test_login_failure", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'login_page'", "trace": "self = <TestCases.Test_YWPT.test_login.TestLogin object at 0x0000024ABF701600>\n\n    def test_login_failure(self):\n        \"\"\"测试登录失败\"\"\"\n>       self.login_page.open()\nE       AttributeError: 'TestLogin' object has no attribute 'login_page'\n\nTestCases\\Test_YWPT\\test_login.py:46: AttributeError"}, "description": "测试登录失败", "start": 1686155712909, "stop": 1686155712909, "uuid": "80ab9c76-a939-41ad-8f47-a6e6793b7f89", "historyId": "84e7926792ca40b2ac442341ce67e51c", "testCaseId": "84e7926792ca40b2ac442341ce67e51c", "fullName": "TestCases.Test_YWPT.test_login.TestLogin#test_login_failure", "labels": [{"name": "parentSuite", "value": "TestCases.Test_YWPT"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-2BK3DCT"}, {"name": "thread", "value": "17844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.Test_YWPT.test_login"}]}